src/Card/Card.js

```diff
@@ -16,6 +18,7 @@ function Card(props) {
         <p>Average Rating: {props.avgRating}</p>
       </article>
     </div>
+    </Link>
```

Comment:
This is a great use of a Link tag encapsulating HTML. Do make sure to indent all of the HTML inside of the Link tag as well.